src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description){
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon(){
+        return (isDone ? "X" : " ");
+    }
+
+    public boolean markAsDone(){
+        isDone = true;
+        return isDone;
+    }
+    public boolean unmarkAsNotDone(){
+        isDone = false;
+        return isDone;
+    }
+}
```

Comment:
Two issues here:
1. Spacing doesn't follow the coding standards
2. please have some comments to all non-trivial classes and/or methods